mne_sandbox/preprocessing/tests/test_sns.py

```diff
@@ -86,5 +89,31 @@ def test_sns():
     # degenerate conditions
     assert_raises(RuntimeError, sns.apply, raw)  # not preloaded
 
+    # Test against NoiseTools
+    rng = np.random.RandomState(0)
+    n_channels = 9
```

Comment:
perhaps add a bad channel here by multiplying the data in one channel by some factor?